client/me/notification-settings/comment-settings/index.jsx

```diff
@@ -68,29 +59,35 @@ const NotificationCommentsSettings = React.createClass( {
 		}
 
 		return ( <p className="notification-settings-comment-settings__placeholder">&nbsp;</p> );
-	},
+	};
 
 	render() {
+		const {
+			path,
+			translate
+		} = this.props;
+
 		return (
 			<Main>
+				<QueryUserDevices />
 				<MeSidebarNavigation />
 				<ReauthRequired twoStepAuthorization={ twoStepAuthorization } />
 
-				<Navigation path={ this.props.path } />
+				<Navigation { ...{ path } } />
 
 				<Card>
 					<FormSectionHeading className="is-primary">
-						{ this.translate( 'Comments on other sites' ) }
+						{ translate( 'Comments on other sites' ) }
 					</FormSectionHeading>
-					<p>{ this.translate( 'Control your notification settings when you comment on other blogs.' ) }</p>
+					<p>{ translate( 'Control your notification settings when you comment on other blogs.' ) }</p>
 					{ this.renderForm() }
```

Comment:
Seems like this would be clearer if inlined instead of as a function call since it's not doing anything complicated in the function.